CHANGELOG.md

```diff
@@ -2,8 +2,13 @@
 
 ## Unreleased
 
+- SONAR-12056 Indicate required fields for forms 
+- SONAR-11959 Colorblind-friendly coverage treemaps
+
 ## 1.0.31
 
+- SONAR-12180 Add a ARIA role to all global success and error messages
+
```

Comment:
What happened here?

Reply:
I forgot to update the Changelog last time 🤷